war/src/main/js/widgets/variables.less

```diff
@@ -1,2 +1,27 @@
+@brightest::#fff;
+@birghtest:#fff;
+@bright:#f9f9f9;
```

Comment:
You are ware that LESS has functions for doing relative colors (brighter, darker etc). Seems like they could be used here.